hello-world/nodejs-cli-app/stop-instance.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Sample code to stop the given instance based on instance OCID and region.
+ * 
+ * Sample Output:
+ * (Prints the instance details)
+ */
+
+const common = require("oci-common");
+const core = require("oci-core");
+
+const adp = new common.ConfigFileAuthenticationDetailsProvider();
+
+const region = common.Region.US_ASHBURN_1;
```

Comment:
Same as above - region shouldn't be hard-coded.

Reply:
Now it's taken from environment variable. But defaults to US_ASHBURN_1 if not given